### PR TITLE
Add showdown (markdown)

### PR DESCRIPTION
**Pull request by adam000**
Use only Showdown for manipulating text. The vanilla version needs a few tweaks before releasing this into the wild, but this version is fine for iFixit use.

### CODE REVIEW

File: nodejs/public/js/ui.messages.js [hunk 1]
```diff
@@ -38,19 +40,8 @@ UI.once('init', function(){
 	}
 	
 	function linkifyAndInsert(el, text){
-		if(Util.Text.linkifyTest(text)){
```
Comment (xiongchiamiov) [inline nodejs/public/js/ui.messages.js:41]: Are we still using `linkifyTest` somewhere?

Comment (adam000) [inline nodejs/public/js/ui.messages.js:41]: We're not using `linkifyTest` or `linkify` anywhere.

Comment (xiongchiamiov) [inline nodejs/public/js/ui.messages.js:41]: Should we perhaps remove them, then?

Comment (adam000) [inline nodejs/public/js/ui.messages.js:41]: e026de5

**Pull request closed without merging**